bridge/dylibs/__init__.py

```diff
@@ -578,7 +616,15 @@ def getplatformpaths(self, libname):
     "win32":    WindowsLibraryLoader
 }
 
-loader = loaderclass.get(sys.platform, PosixLibraryLoader)()
+def get_library_loader():
+    loader = loaderclass.get(sys.platform, PosixLibraryLoader)()
+
+    if platform.architecture()[0].startswith('64'):
+        loader = Windows64LibraryLoader()
+
+    return loader
+
+loader = get_library_loader()
```

Comment:
For Oculus I don't need ctypes.windll for either the 32 or the 64 bits version (it can be either cdll or windll). That said, if what is required for VIVE is different from the 32 to the 64 bits version I recommend the following code instead:

```
diff --git a/python/bridge/dylibs/__init__.py b/python/bridge/dylibs/__init__.py
index b4a41f0..23c7e28 100644
--- a/python/bridge/dylibs/__init__.py
+++ b/python/bridge/dylibs/__init__.py
@@ -531,6 +531,8 @@ class _WindowsLibrary(object):
 
 class WindowsLibraryLoader(LibraryLoader):
     name_formats = ["%s.dll", "lib%s.dll", "%slib.dll"]
+    is_64 = platform.architecture()[0].startswith('64')
+    DLL = ctypes.windll if is_64 else ctypes.cdll
 
     def load_library(self, libname):
         try:
@@ -540,14 +542,14 @@ class WindowsLibraryLoader(LibraryLoader):
             if os.path.sep not in libname:
                 for name in self.name_formats:
                     try:
-                        result = getattr(ctypes.cdll, name % libname)
+                        result = getattr(DLL, name % libname)
                         if result:
                             break
                     except WindowsError:
                         result = None
             if result is None:
                 try:
-                    result = getattr(ctypes.cdll, libname)
+                    result = getattr(DLL, libname)
                 except WindowsError:
                     result = None
             if result is None:
```